docs/content/v0.20/packaging.md

```diff
@@ -9,48 +9,135 @@ Note that to _use_ KubeStellar, all of the repos in this graph are involved.
+    osa_hc_src -.-> osa_ctr_image
+    osa_hc_repo -.-> osa_ctr_image
+```
+
+The dashed dependencies are at run time, not build time.
+
+"OSA" is OCM Status Addon
+
+### OCM status addon container image
+
+There is a container image at [ghcr.io/kubestellar/ocm-status-addon](https://github.com/orgs/kubestellar/packages/container/package/ocm-status-addon).
+
+The container image is built and published using `make`.  The
+following `make` variables are relevant.
+
+- `KO_DOCKER_REPO`, defaults to `ghcr.io/kubestellar`
+- `CMD_NAME`, defaults to `ocm-status-addon`
+- `IMAGE_TAG`, defaults to `0.2.0-alpha.1`
+- `IMG`, defaults to `${KO_DOCKER_REPO}/${CMD_NAME}:${IMAGE_TAG}`
+
+`make ko-build-push` will build and publish the container image at
+`${IMG}`. This will be a multi-platform manifest, referencing images
+for "platforms" `linux/amd64` and `linux/arm64`. This will not work if
+you have `podman` aliased to `docker`.
+
+`make ko-local-build` will build the same multi-platform manifest but
+not push it, only leave it among your Docker images. This also will
+not work if you have `podman` aliased to `docker`.
+
+
+### OCM status addon Helm chart
+
+The operator is delivered by a Helm chart at [ghcr.io/kubestellar/ocm-status-addon-chart](https://github.com/orgs/kubestellar/packages/container/package/ocm-status-addon-chart).
 
-The [status-addon](https://github.ibm.com/dettori/status-addon) repo is the source of a RedHat-style operator.
+The chart is built and published using `make`. The following `make`
+variables are relevant.
 
-The operator is delivered by a Helm chart at `quay.io/pdettori/status-addon-chart`. Chart versioning? How is the chart built, pushed?
+- `KO_DOCKER_REPO`, defaults to `ghcr.io/kubestellar`
+- `CMD_NAME`, defaults to `ocm-status-addon`
+- `IMAGE_TAG`, defaults to `0.2.0-alpha.1`
+- `IMG`, defaults to `${KO_DOCKER_REPO}/${CMD_NAME}:${IMAGE_TAG}`
 
-There is also a container image involved.
+`make chart`: builds the local copy of the chart from local sources,
+using `kustomize`. When using the default values of the `make`
+variables, the built chart references the container image
+`ghcr.io/kubestellar/ocm-status-addon:0.2.0-alpha.1`. TODO: describe
+how the chart contents and destination depend on the `make` variables.
+
+`make chart-push` (depends on the `make` target `chart`): pushes the
+local copy of the chart to the OCI repository
+[ghcr.io/kubestellar/ocm-status-addon-chart](https://github.com/orgs/kubestellar/packages/container/package/ocm-status-addon-chart). How
+is the OCI tag of the chart controlled?
+
+The following versions exist.
+
+| OCI tag = version in chart | appVersion in chart | referenced container tag |
+| -------------------------- | ------------------- | ------------------------ |
+| 0.2.0-alpha.1              | 0.2.0-alpha.1       | 0.2.0-alpha.1 |
+
+Note: the Chart.yaml in github uses just "0.2.0" for the version and appVersion in the chart. TODO: bug?
```

Comment:
indeed

Reply:
I think this bug is item (4) in https://github.com/kubestellar/ocm-status-addon/issues/11